OrbitGl/CaptureSerializer.cpp

```diff
@@ -240,17 +248,25 @@ ErrorMessageOr<void> CaptureSerializer::Load(std::istream& stream) {
     ERROR("%s", error_message);
     return ErrorMessage(error_message);
   }
-  ProcessCaptureData(capture_info);
+  CaptureData capture_data = GenerateCaptureData(capture_info);
 
   // Timers
   TimerInfo timer_info;
   while (ReadMessage(&timer_info, &coded_input)) {
-    time_graph_->ProcessTimer(timer_info);
+    if (timer_info.function_address() > 0) {
+      const FunctionInfo& func =
+          capture_data.selected_functions().at(timer_info.function_address());
+      capture_data.process()->GetFunctionFromAddress(timer_info.function_address());
```

Comment:
```suggestion

```

Reply:
I guess that would have been caught by a [[nodiscard]]

Reply:
Thanks